packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { Suspense, useEffect, useState } from 'react';
+
+import useIsBrowser from './useIsBrowser';
+import useIsIntersecting from './useIsIntersecting';
+
+import type { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+const Overlay = () => (
+  <div
+    data-mux-player-react-lazy-overlay
+    style={{
+      position: 'absolute',
+      inset: 0,
```

Comment:
Considering we're using `aspect-ratio`, I'd say we're safe to use `inset`, too? 

https://caniuse.com/?search=inset